Pipfile.lock

```diff
@@ -148,6 +148,14 @@
             ],
             "version": "==3.0.0"
         },
+        "django-tailwind": {
```

Comment:
Please also add this dependency to the `Pipfile` (Technically, it will be installed if it's only contained in `Pipfile.lock`, but the next time someone changes the `Pipfile`, this dependency will be kicked out from `Pipfile.lock`).

Reply:
I added it to `setup.py` and ran `./dev-tools/update_dependencies.sh` which added it to `Pipfile.lock`. When to use `Pipfile` vs `setup.py`?

Reply:
Ah, I didn't see the entry in `setup.py`. The difference is whether the dependency is required on the production server or not ([see docu](https://integreat.github.io/integreat-cms/virtualenv.html#add-dependencies)).
In thought that the `tailwind build` step produces output which can be served as static files, which means that we only need this dependency on dev environments (=> `Pipfile`)?

Reply:
Got it, thanks 😄 prod only needs the css file. I'll change it accordingly.